tests/ct_connection.c

```diff
@@ -401,6 +401,9 @@ void test_async_password_wrong(void) {
     assert(acc->err == REDIS_ERR_OTHER);
     assert(strcmp(acc->errstr, "slotmap not available") == 0);
 
+    /* Since the owner of the slot is unknown the redisClusterAsyncCommand will
+       initite a slotmap update. This update will also get a WRONGPASS error. */
+
     event_base_dispatch(base);
```

Comment:
... and after WRONGPASS, it disconnects which removes the fd from the event loop which exits?

I am slightly confused about what happens behind the scenes.

Reply:
Reverted this change. There is no need for an eventloop to test the WRONGPASS scenario here.